src/specimens/Table.js

```diff
@@ -40,13 +40,25 @@ function getStyle(theme) {
   };
 }
 
-const Cell = (value, id, style, heading) => {
-  const content = typeof value === 'string'
-    ? renderMarkdown({text: value})
-    : value || <p style={{opacity: 0.2}}>—</p>;
+const Cell = ({value, style, heading}) => {
+  let content;
+  if (typeof value === 'string') {
+    content = renderMarkdown({text: value.toString()});
+  } else if (value === void 0) {
```

Comment:
`void 0` vs `undefined`. We currently use both in this project, but should settle on one or the other. I prefer `undefined`.